functional_test.sh

```diff
@@ -35,23 +39,36 @@ docker run \
     -classpath web/target/graphhopper-web-1.0-SNAPSHOT.jar com.graphhopper.http.GraphHopperApplication gtfs_links default_gh_config.yaml
 
 # Run server in background
-docker run --rm  --name functional_test_server -p 50051:50051 -p 8998:8998 -v "$TMPDIR:/graphhopper/transit_data/" \
+docker run --rm --name functional_test_server -p 50051:50051 -p 8998:8998 -v "$TMPDIR:/graphhopper/transit_data/" \
     "$DOCKER_IMAGE_TAG" &
 
 echo "Waiting for graphhopper server to start up"
-sleep 30
+sleep 30 # should we implement a trie mechanism with time threshold instead of sleep?
+
+# greb the server ip:
+SERVER= ` docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' functional_test_server }}`
```

Comment:
Small style thing - can we use `$(docker inspect ...)` instead of backticks? I prefer using that because it composes better. There are also a couple of extraneous spaces in here, and I think there's a typo in the comment above